src/style.scss

```diff
@@ -20,9 +20,37 @@ $auro-inset-directions: '';
   cursor: pointer;
 }
 
+[data-trigger-placement]::slotted(*:hover) {
```

Comment:
I am not sure this update does what you think it is doing. Yes, it corrected the flutter I saw, but I don't think that this is actually addressing the problem that it was supposed to do. 

To make this simpler, I am suggesting some updates to how this element works that will make this easier.